src/infinite-scroll.coffee

```diff
@@ -146,16 +178,29 @@ mod.directive 'infiniteScroll', ['$rootScope', '$window', '$interval', 'THROTTLE
     scope.$watch 'infiniteScrollUseDocumentBottom', handleInfiniteScrollUseDocumentBottom
     handleInfiniteScrollUseDocumentBottom scope.infiniteScrollUseDocumentBottom
 
+    # allow to use promises to load more data.
+    # This option will wait for the promise to be resolve before permitting
```

Comment:
"This option will wait for the promise to resolve before awaiting a new one"

or, "This option will await each promise before awaiting the next promise."

Or "When true, ngInfiniteScroll will await each promise serially"